matrix-client.el

```diff
@@ -371,7 +376,9 @@ and password."
         (add-hook 'buffer-list-update-hook #'matrix-client-buffer-list-update-hook 'append 'local))
       (erase-buffer)
       (matrix-client-render-message-line room-obj)
-      (matrix-client-insert-last-seen-overlay))
+      (matrix-client-insert-last-seen-overlay)
+      (when matrix-client-use-tracking
+        (tracking-mode)))
```

Comment:
Quick question I just thought of: If tracking-mode is already activated, like from circe, will this deactivate it?

Reply:
tracking mode is a buffer-local minor mode, so it should come turned off by default, but it's a good idea to be safe anyway, I'll add a commit forcing it on :)

Reply:
Ah, nevermind, I think it's fine:

> When called from Lisp, the mode command toggles the mode if the
argument is ‘toggle’, disables the mode if the argument is a
non-positive integer, and enables the mode otherwise (including
if the argument is omitted or nil or a positive integer).

Reply:
Actually, I was wrong, it is a global minor mode, it dosen't matter though either way :P 

https://github.com/jorgenschaefer/circe/wiki/Tracking